electron/main.js

```diff
@@ -402,25 +415,88 @@ const createAgentActivityWindow = async () => {
   return agentWindow;
 };
 
+// Create SSL certificate for the backend
+function createSSLCertificate() {
+  try {
+    logger.electron('Creating SSL certificate...');
+
+    const certDir = path.join(paths.dotOperateDirectory, 'ssl');
+    if (!fs.existsSync(certDir)) {
+      fs.mkdirSync(certDir, { recursive: true });
+    }
```

Comment:
it seems, a new certificate will be created on each launch, is that so? I don't see any checks for files existence

Reply:
It is planned to use new certificates on each run so that the don't expire. We don't check their existence and they are simply overwritten every time the app starts.